NameCorrect.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+public class NameCorrect {
+    public static void main(String[] args) {
+        try{
+        int vowelCount = 0;
+        int consonantCount = 0;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the string : ");
+        String input = sc.nextLine();
+        System.out.println("\nCorrect name would be :");
+
+        String[] name = input.split(" ");
+        for(int i=0;i<=name.length-1;i++) {
```

Comment:
Is it that required, I suggest just pick length of an array and use charAt() function

Reply:
But by charAt(), how will I be able to select the last name.
Also, it would not be feasible if someone inputs a longer name like 4-5 words.